bindings/java/src/convert.rs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use jni::errors::Result;
+use jni::objects::{JMap, JObject, JString, JThrowable, JValue};
+use jni::JNIEnv;
+use opendal::ErrorKind;
+use std::collections::HashMap;
+
+pub(crate) fn error_to_error<E>(error: E) -> opendal::Error
```

Comment:
The naming doesn't LGTM. And I don't like to implement functions upon `Into<anyhow::Error> + ToString` which can be used in wrong easily.

How about implement `parse_jni_error(error: jni::errors::Error) -> opendal::Error`?